tests/installation/test_installer.py

```diff
@@ -76,6 +76,18 @@ def load(cls, env):
         return cls()
 
 
+@pytest.fixture
+def working_directory():
```

Comment:
Can we move this into `tests/conftest.py` and rename it into something referring poetry source root?

Reply:
@abn thanks for the feedback!  i moved it here: https://github.com/python-poetry/poetry/pull/3231/commits/87556f8d3f842b7872b1812fad4da19975df432a

I also used it for test_exporter https://github.com/python-poetry/poetry/pull/3231/commits/584a9e648d7ac6e1e077b04272e095947d377b7a

Reply:
I also moved the `mock_path_cwd` to conftest. wdyt? thank you

Reply:
I suspect that should be fine; althought would like to double check if other test cases were mocking cwd. I believe there are test cases that update cwd to tbe a temporary project directory - just need to make sure that this fixture does not override those in the future. It might just be working right now due to the order the fixtures are loaded. Would be great if you can cross check that too.

Reply:
@abn you're right, there are cases in other files that patch the cwd too. It will work with the current setup because it will be loaded first and then overwritten later. But I reverted it nevertheless, because it will be required by a just few and not all test suites anyway. thank you

Reply:
Thank you for checking up on that.